integration/sawtooth_integration/tests/burrow_evm_intkey_test.go

```diff
@@ -89,8 +106,8 @@ func TestIntkey(t *testing.T) {
 
 }
 
-func exec(t *testing.T, client *burrow_evm_client.Client, priv, to, data []byte) {
-  _, err := client.Exec(priv, to, data, 1000)
+func exec(t *testing.T, c *client.Client, priv, to, data []byte, nonce uint64) {
```

Comment:
See my overall comment about naming.  This line was merely an instance where you were removing the context of a variable for no gain in readability.